packages/SketchMorph2-Core.package/M2GripMorph.class/instance/updatePosition.st

```diff
@@ -0,0 +1,5 @@
+initialization
```

Comment:
Since `updatePosition` gets called during the lifetime of the object, I think this category no longer applies.